modyn/tests/config/test_config_integrity.py

```diff
@@ -0,0 +1,55 @@
+from pathlib import Path
+
+import pytest
+from modyn.config import read_modyn_config, read_pipeline
+from modynclient.config import read_client_config
+
+"""Directories can either be file paths or directories.
+If they are directories, all yaml files in the directory will be used."""
+
+MODYN_PIPELINE_CONFIG_PATHS: list[str] = [
+    "modynclient/config/examples/dummy.yaml",
```

Comment:
These paths are probably based on the working directory. Does this also work when I run pytest from some other directory than the project root? I think typically we do something like in the integration tests:

```
SCRIPT_PATH = pathlib.Path(os.path.realpath(__file__))
CONFIG_FILE = SCRIPT_PATH.parent.parent.parent / "modyn" / "config" / "examples" / "modyn_config.yaml"
````

to infer the actual project directory and avoid any relative path fun

Reply:
Ah yes, sure :)
btw. pathlib supports access parents via `path.parents[depth]`

Reply:
ah thanks, did not know this